app/views/api/v1/accounts/articles/_article.json.jbuilder

```diff
@@ -0,0 +1,10 @@
+json.id article.id
+json.category_id article.category_id
+json.title article.title
+json.content article.content
+json.description article.description
+json.status article.status
+json.account_id article.account_id
+json.portal article.portal
+json.views article.views
+json.author article.author
```

Comment:
This would send all attributes of the user, right? I think we should use the agent.json.jbuilder view here.